contracts/F3Parameters.sol

```diff
@@ -18,8 +18,10 @@ contract F3Parameters is Ownable {
     uint64 private _activationEpoch;
 
     /// @dev The data associated with the manifest for the parameters.
+    ///      This is intended to contain DEFLATE compressed JSON data that must be decompressed before parsing by consumers.
```

Comment:
Wanted to give signal to someone inspecting messages on chain how to parse this parameter.